### PR TITLE
Read support for steam oven/microwave combination

### PR DESCRIPTION
**Pull request by DAccord**
Adding read support for steam oven/microwave combination, i.e. Miele integration will now show correct program names when using the steam oven/microwave combination. Write support not included.
Tested with Miele DGM 7440.

### CODE REVIEW

File: custom_components/miele/translations/en.json [hunk 1]
```diff
@@ -284,7 +284,423 @@
...
+          "sous_vide": "Sous-vide",
+          "eco_steam_cooking": "Eco Steam cooking",
+          "rapid_steam_cooking": "Rapid steam cooking",
+          "descale": "Descale",
+          "menu_cooking": "Menu cooking",
+          "reheating_with_steam": "Reheating with steam",
+          "defrosting_with_steam": "Defrosting with steam",
+          "blanching": "Blanching",
+          "bottling": "Bottling",
+          "heat_crockery": "Heat crockery",
```
Comment (astrandb) [inline custom_components/miele/translations/en.json:301]: Duplicate
```suggestion
```

File: custom_components/miele/translations/en.json [hunk 2]
```diff
@@ -284,7 +284,423 @@
...
+          "savoy_cabbage_cut": "Savoy cabbage (cut)",
+          "courgette_sliced": "Courgette (sliced)",
+          "courgette_diced": "Courgette (diced)",
+          "snow_pea": "Snow pea",
+          "perch_whole": "Perch (whole)",
+          "perch_fillet_2_cm": "Perch (fillet, 2 cm)",
+          "perch_fillet_3_cm": "Perch (fillet, 3 cm)",
+          "gilt_head_bream_whole": "Gilt-head bream (whole)",
+          "gilt_head_bream_fillet": "Gilt-head bream (fillet)",
+          "codfish_piece": "Codfish (piece)",
```
Comment (astrandb) [inline custom_components/miele/translations/en.json:454]: Duplicate
```suggestion
```

File: custom_components/miele/translations/en.json [hunk 3]
```diff
@@ -284,7 +284,423 @@
...
+          "codfish_fillet": "Codfish (fillet)",
```
Comment (astrandb) [inline custom_components/miele/translations/en.json:455]: Duplicate
```suggestion
```

File: custom_components/miele/translations/en.json [hunk 4]
```diff
@@ -284,7 +284,423 @@
...
+          "knuckle_of_pork_fresh": "Knuckle of pork (fresh)",
+          "knuckle_of_pork_cured": "Knuckle of pork (cured)",
+          "pork_tenderloin_medaillons_3_cm": "Pork tenderloin (medaillons, 3 cm)",
+          "pork_tenderloin_medaillons_4_cm": "Pork tenderloin (medaillons, 4 cm)",
+          "pork_tenderloin_medaillons_5_cm": "Pork tenderloin (medaillons, 5 cm)",
+          "pumpkin_soup": "Pumpkin soup",
+          "meat_with_rice": "Meat with rice",
+          "beef_casserole": "Beef casserole",
+          "risotto": "Risotto",
+          "risotto": "Risotto",
```
Comment (astrandb) [inline custom_components/miele/translations/en.json:537]: Duplicate
```suggestion
```

File: custom_components/miele/translations/en.json [hunk 5]
```diff
@@ -284,7 +284,423 @@
...
+          "veal_fillet_medaillons_3_cm": "Veal fillet (medaillons, 3 cm)",
+          "goulash_soup": "Goulash soup",
+          "dutch_hash": "Dutch hash",
+          "stuffed_cabbage": "Stuffed cabbage",
+          "beef_tenderloin": "Beef tenderloin",
+          "beef_tenderloin_medaillons_1_cm_steam_cooking": "Beef tenderloin (medaillons, 1 cm, steam cooking)",
+          "beef_tenderloin_medaillons_2_cm_steam_cooking": "Beef tenderloin (medaillons, 2 cm, steam cooking)",
+          "beef_tenderloin_medaillons_3_cm_steam_cooking": "Beef tenderloin (medaillons, 3 cm, steam cooking)",
+          "silverside_5_cm": "Silverside (5 cm)",
+          "silverside_7.5_cm": "Silverside (7.5 cm)",
```
Comment (astrandb) [inline custom_components/miele/translations/en.json:518]: No period in key
```suggestion
          "silverside_7_5_cm": "Silverside (7.5 cm)",
```

**Review by astrandb — CHANGES_REQUESTED** [review]
Great job. Where did you find all these strings?
A handful of minor comments...

**Comment by DAccord** [comment]
Took me a 2-3 afternoons to try out all the programs on my DGM 7440 and record the IDs and corresponding names.

File: custom_components/miele/translations/en.json [hunk 6]
```diff
@@ -533,8 +529,6 @@
           "pumpkin_soup": "Pumpkin soup",
           "meat_with_rice": "Meat with rice",
           "beef_casserole": "Beef casserole",
-          "risotto": "Risotto",
-          "risotto": "Risotto",
```
Comment (astrandb) [inline custom_components/miele/translations/en.json]: One risotto should be retained.

**Review by astrandb — CHANGES_REQUESTED** [review]
Just a little one left.
Note that en.json will be sorted in alphabetic order when up/downloaded to Lokalise.

File: custom_components/miele/translations/en.json (hunk 6 above)
Comment (DAccord) [reply custom_components/miele/translations/en.json]: Not sure why this got lost ...

**Comment by astrandb** [comment]
Thank you @DAccord

**Review by astrandb — APPROVED** [review]

**Pull request merged**